### ISSUE TITLE
Page Layouts : The focusable element for each page layout does not have a proper role 

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5903]
## Bug Description

as reported by @barklund 
The focusable element for each page layout does not have a proper role - it’s just a div. This should probably be a button.

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by BrittanyIRL** [comment]
Fixed with #5876
